### PR TITLE
projectUser assignment endpoints

### PR DESCRIPTION
**Pull request by dheniges**
So I nuked the idea of these endpoints returning user objects now that there's data associated with the join (that data being the project role assigned currently, and maybe more in the future).

If we do want to get project users, we can add new endpoints at the old url of /projects/:id/users and/or support the include? syntax discussed on an endpoint.

I also blew up the husky hook because I'm the only one that runs it on web and it blew up tons of files and rewrote Jay's css which made him very happy

### CODE REVIEW

File: api/src/businesstime/projectuser.test-i.js [hunk 1]
```diff
@@ -3,7 +3,9 @@ import ProjectFactory from '../db/__testSetup__/factories/project'
 import UserFactory from '../db/__testSetup__/factories/user'
 import initializeTestDb, { clearDb } from '../db/__testSetup__/initializeTestDb'
 import constants from '../db/__testSetup__/constants'
-import { PUBLIC_FIELDS as USER_PUBLIC_FIELDS } from './user'
+
+// TODO: in BusinessTime, define and filter by public fields somewhere
```
Comment (jjidt) [inline api/src/businesstime/projectuser.test-i.js]: is this something we need to wait on?

File: api/src/controllers/projectUsers.js [hunk 2]
```diff
@@ -72,39 +72,39 @@ const getProjectUser = async function(req, res) {
...
   try {
-    await Promise.all(
-      body.users.map(u => BusinessProjectUser.deleteByUserIdForProject(u.id, projectId, orgId))
+    const projectUser = await BusinessProjectUser.updateProjectUserById(
+      id,
+      body.roleId,
+      orgId
     )
-    res.status(204).send()
+    res.status(201).json({ data: projectUser })
```
Comment (jjidt) [inline api/src/controllers/projectUsers.js]: should this be a 200 code on the update?

**Review by jjidt — COMMENTED** [review]
Just a couple questions, but overall this seems like a good structure 💯

**Comment by dheniges** [comment]
this is ready for re-review

**Review by jjidt — APPROVED** [review]
👍

**Pull request merged**